app/controllers/admin/participants/change_name_controller.rb

```diff
@@ -22,9 +26,19 @@ def update
 
   private
 
+    def check_gai_status
+      if @participant_profile.user.get_an_identity_id.present?
+        flash[:alert] = "GAI users cant access this page"
```

Comment:
Might want to spell this out more clearly, support might not know our acronyms.

"Get an Identity registered users must change their names via the Get an Identity service." perhaps?